src/app/zones/locales/en-us/index.yaml

```diff
@@ -62,8 +66,8 @@ zones:
             kdsDeltaEnabled: true
         connectDescription: 'Next, download Kuma and connect the Zone'
         connectCommand: |
-          (export VERSION=0.0.0-preview.vabf3cdb72; curl -L https://kuma.io/installer.sh | sh - \
-            && kuma-$VERSION/bin/kuma-cp run -c config.yaml)
+          curl -L https://kuma.io/installer.sh | VERSION=0.0.0-preview.vabf3cdb72 sh - \
+            && kuma-0.0.0-preview.vabf3cdb72/bin/kuma-cp run -c config.yaml
```

Comment:
Not sure how worth it it would be but should we be passing these in as a variable sourced from either `env` or `t`, do we have these values already in `env`? I think we have version, but maybe not the hash over here. The hash might be worth adding over here also? We'd probably use it for other things.

Anyway no action required, just half thinking out loud half asking what you think.

Reply:
That’s not worth the effort because this is a temporary measure as far as I understand it. The long term variant shouldn’t include a specific version at all.

For the time being, this being here is extra beneficial because it means other folks can update it in the language files.

Reply:
Gotcha, 

> For the time being, this being here is extra beneficial because it means other folks can update it in the language files.

Well ideally they wouldn't have to, it would automatically change.

Anyway, all good, this is tmp so 🤷‍♂️